pkg/stanza/fileconsumer/metadata.yaml

```diff
@@ -9,8 +9,31 @@ status:
     active: [andrzej-stencel]
     emeritus: [djaglowski]
 
+attributes:
+  file_path:
+    description: The path of the file being monitored
+    type: string
+
 telemetry:
   metrics:
+    fileconsumer_file_offset:
+      description: Current read offset in files being monitored. This value may reset when files are rotated.
+      unit: "By"
+      enabled: true
```

Comment:
Can we land this without enabling those metrics by default, and turn them on by default after they've had time to bake?